schemas/application.schema.json

```
@@ -388,14 +388,205 @@
               null
             ]
           },
-          "projectTotalCosts": {
-            "$id": "#/properties/application/properties/projectTotalCosts",
-            "type": "integer",
-            "title": "project total cost (in £)",
-            "default": 0,
-            "examples": [
-              20000
-            ]
+          "projectCosts": {
+            "$id": "#/properties/application/properties/projectCosts",
+            "type": "array",
+            "title": "The Projectcosts Schema",
+            "items": {
+              "$id": "#/properties/application/properties/projectCosts/items",
+              "type": "object",
+              "title": "The Items Schema",
+              "required": [
+                "costType",
+                "costDescription",
+                "costAmount"
+              ],
+              "properties": {
+                "costType": {
+                  "$id": "#/properties/application/properties/projectCosts/items/properties/costType",
+                  "type": "string",
+                  "title": "The Costtype Schema",
+                  "default": "",
+                  "examples": [
+                    "new-staff"
+                  ],
+                  "enum": ["new-staff", "professional-fees", "purchase-price-of-heritage-items", "repair-and-conservation-work", "event-costs", "digital-outputs", "equipment-and-materials-including-learning-materials", "training-for-staff", "training-for-volunteers", "travel-for-staff", "travel-for-volunteers", "expenses-for-staff", "expenses-for-volunteers", "publicity-and-promotion"],
```

```suggestion
                  "enum": ["new-staff", "professional-fees", "recruitment", "purchase-price-of-heritage-items", "repair-and-conservation-work", "event-costs", "digital-outputs", "equipment-and-materials-including-learning-materials", "training-for-staff", "training-for-volunteers", "travel-for-staff", "travel-for-volunteers", "expenses-for-staff", "expenses-for-volunteers", "other", "publicity-and-promotion", "evaluation", "contingency"],
```

This will add the missing cost headings.